test/integration/image-component/basic/pages/client-side.js

```diff
@@ -9,7 +9,7 @@ const ClientSide = () => {
       <Image
         id="basic-image"
         src="foo.jpg"
-        lazy={false}
+        lazy="eager"
```

Comment:
```suggestion
        loading="eager"
```

Same for the other tests